lib/rubocop/cop/style/string_methods.rb

```diff
@@ -0,0 +1,52 @@
+# encoding: utf-8
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop enforces the use of consistent method names
+      # from the String class.
+      class StringMethods < Cop
```

Comment:
Please extract code that's the same in `CollectionMethods` to a new module, called`MethodPreference` or `PreferredMethods` under `cop/mixin`.